guides/vim-cheatsheet.rst

```diff
@@ -0,0 +1,444 @@
+Vim Commands Cheat Sheet
```

Comment:
Very funny … sphinx is still compiling the emacs cheat sheet.. more news soon.  ;-)